src/components/_app-bar.scss

```
@@ -0,0 +1,110 @@
+/*doc
+---
+title: App Bar
+name: app-bar
+category: Components
+---
+
+App Bar には現在の画面に関連する情報とアクションが表示されます。`.ncgr-app-bar`に`-active`を付与することでアクティブな状態になります。
+
+`leading`、`title`、`actions`の３つの要素がありますが、`leading`は閉じる、戻るなどの画面のナビゲーションを操作するボタンを置き、`actions`にはオプションや通知欄など、画面内のアクションを促すボタンを設置してください。
+
+また、Appbarの領域をもたずにモーダルを閉じるためのボタン用に、`ncgr-app-bar__close`が用意されています。
+
+```example
+<div class="ncgr-app-bar -active">
+  <div class="ncgr-app-bar__leading">
+    <div class="ncgr-app-bar__icon ncgr-typography-label">
+      ×
+    </div>
+  </div>
+  <div class="ncgr-app-bar__title">
+    メニュー
+  </div>
+  <ul class="ncgr-app-bar__actions">
+    <li>
+      <i class="fas fa-ellipsis-v ncgr-typography-label"></i>
+    </li>
+  </ul>
+</div>
+```
+
+*/
+
+.ncgr-app-bar {
+  width: 0;
+  height: 0;
+  background-color: var(--color-secondary-grouped-background);
+  display: flex;
+  align-items: flex-start;
+  box-sizing: border-box;
+  z-index: 2147483646;
+  &.-transparent {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: transparent;
+    .ncgr-app-bar__leading {
+      margin: 0.5rem auto auto 0.5rem;
+    }
+    .ncgr-app-bar__icon {
+      width: 2.5rem;
+      height: 2.5rem;
+      background:rgba(0, 0, 0, 0.3);
```

[IMO] もうちょい濃いほうがよみやすそう...？

```suggestion
      background:rgba(0, 0, 0, 0.4);
```